runtime/gc_base/RootScanner.hpp

```diff
@@ -198,7 +198,7 @@ class MM_RootScanner : public MM_BaseVirtual
 		_scanningEntity = scanningEntity;
 		
 		if (_extensions->rootScannerStatsEnabled) {
-			OMRPORT_ACCESS_FROM_OMRVM(_omrVM);
+			OMRPORT_ACCESS_FROM_OMRVM(_javaVM->omrVM);
```

Comment:
could use PORT_ACCESS_FROM_JAVAVM

Reply:
I also thought so... Appears to be not really. First macro leads to OMR Port, second one for J9 Port. Using it causes compile errors like:
```
     [exec] "gc_base/RootScanner.hpp", line 205: error: identifier "privateOmrPortLibrary"
     [exec]           is undefined
     [exec]      _entityStartScanTime = privateOmrPortLibrary->time_hires_clock(privateOmrPortLibrary);
     [exec]                             ^
```